src/Model/Scope/Condition.php

```diff
@@ -195,8 +195,22 @@ public function toQueryArguments(): array
                     }
 
                     foreach (array_reverse($refModels) as $refModel) {
-                        if ($field === '#') {
-                            $field = $value ? $refModel->action('count') : $refModel->action('exists');
+                        // # is alias for count(*)
+                        if ($field === '#' || $field === 'count(*)') {
+                            $field = $refModel->action($value ? 'count' : 'exists');
+
+                            continue;
+                        }
+
+                        $matches = [];
+                        // support for conditions on aggregates
+                        if (is_string($field) && preg_match('~^(\w+)\(([\w *]*)\)$~', $field, $matches)) {
+                            $field = $refModel->action('fx', array_slice($matches, 1));
+
+                            if (!$value) {
```

Comment:
is'n missing value handled by dsql?

is this 1:1 sql equivalent? What about -1?

Reply:
Well it is handled by sql...
condition is `... and (select sum("price") from "ticket" ...)`

Will remove the fallback in php

Reply:
:)

Reply:
All updated.
I think Space X can now safely use atk4/data for crew dragon - out of the box 😆